object/object.go

```diff
@@ -231,6 +231,29 @@ func (m *Map) KV() []KV {
 	return m.kv
 }
 
+var (
+	KeyKey   = String{Value: "key"}
+	ValueKey = String{Value: "value"}
```

Comment:
```suggestion
	Key   = String{Value: "key"}
	Value = String{Value: "value"}
```

Or a common prefix


```suggestion
	KeyKey   = String{Value: "key"}
	KeyValue = String{Value: "value"}
```

Reply:
can't use the first one (that was what I had) because it's same name as the fields (well you can but it looks weird to have Key: Key, though that's what I wrote at first)

Reply:
and it's keys so it's the value's key, I think a suffix is clear ValueKey vs KeyValue sounds like a Value

Reply:
OK 👍